### ISSUE TITLE
Potential Bug with SVG Upload Action

### ISSUE THREAD

**Issue by calebsylvest** [artemartemov/sanity-plugin-inline-svg-upload-preview#3]
Howdy,

I've run into a potential bug with the SVG Uploader after the last couple of Sanity Studio updates where the **Upload SVG** button is no longer clickable. Previously uploaded SVGs can still be removed, but once gone you can't upload a new one.

I have not been able to track down the source of the issue (which may be beyond my capabilities) but I was able to create a hack with CSS to show the default browser **Choose File** button and make use of this plugin with that method.

### Question
Is anyone else experiencing this issue with the latest Sanity updates? @artemartemov 

### Console Error Messages
![image](https://user-images.githubusercontent.com/4457883/92006886-e4110d00-ed0a-11ea-8ac8-6ae987ca28c2.png)

### My CSS Hack
<img width="1438" alt="OWYN_–_Sanity" src="https://user-images.githubusercontent.com/4457883/91925798-695cd900-ec9b-11ea-8655-3a8fe9d2ab86.png">

### Sanity Studio Versions
![image](https://user-images.githubusercontent.com/4457883/92006363-3f8ecb00-ed0a-11ea-858e-d99ba4b602db.png)

**Comment by calebsylvest** [comment]
Here is the simple fix I came up with:

```
.svgPreviewBackground input[type=file] {
	cursor: pointer;
	opacity: 0;
	width: 100%;
	position: absolute;
	top: 0;
	right: 0;
	bottom: 0;
	left: 0;
	z-index: 1;
}
```

This will make the click area fill the entire upload zone, appear invisible, and ensure it is on the z-index above the faux upload button.

**Comment by calebsylvest** [comment]
Added a PR to fix these issues. Tested it locally and made another update.

**Comment by artemartemov** [comment]
fixed with #5
